From 2b8c1f1e2b75e7eb6f3e79c294a74e547960de76 Mon Sep 17 00:00:00 2001
From: Ryo Nakano <ryonakaknock3@gmail.com>
Date: Sat, 29 Jun 2024 08:52:04 +0000
Subject: [PATCH] lib: Include stylesheet in the library

Export the stylesheet resource as a part of the library
so that we can use libswitchboard as a widget library
from other apps, not from Switchboard plugs.
---
 lib/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/meson.build b/lib/meson.build
index 5f5f733..922766b 100644
--- a/lib/meson.build
+++ b/lib/meson.build
@@ -26,6 +26,7 @@ libswitchboard_lib = library('switchboard-3',
     'SettingsSidebarRow.vala',
     'SettingsSidebar.vala',
     config_header,
+    stylesheet_resource,
     dependencies: [libswitchboard_deps, config_vapi],
     vala_header: 'switchboard.h',
     soversion: '0',
-- 
2.43.0

